### DISCUSSION TITLE
Has the mjai.app stopped now?

### DISCUSSION THREAD

**Discussion by RuBP17** [smly/mjai.app#18] [Q&A] [answered]
I saw that the last game in the mjai.app is in the Feb 2023, so I doubt has the project stopped?

**Comment by smly** [comment, marked as answer]
Thanks for your interest. Right now I am suspending the mjai.app project, but I am preparing to resume it!

Unfortunately, I will be traveling for a week to attend an international conference. I will be ready to resume next month.

Before resuming the project, I plan to do the following:

* There is a bug in the score calculation. Fix the bug.
* Make it easier for developers to verify that the Agent works without errors. Provide an evaluation method by implementing an open source simulator.
* Make the games run in batch processing only on Saturdays and Sundays. For saving my money 🤑 
* (Optional) Stop using Twitter authentication, and use Auth0 for alternative third-party authentication.

Thanks!

**Comment by smly** [comment]
It is still under development, but I have fixed a bug in the game simulator and made it open source.
https://github.com/smly/mjai.app

Currently, I have just completed adjusting our matching algorithm. Now I will begin updating the web application and creating documentation. Please wait for a moment!
